rgw/v2/tests/s3_swift/test_indexless_buckets.py

```diff
@@ -41,8 +41,9 @@ def test_exec(config):
     io_info_initialize = IOInfoInitialize()
     basic_io_structure = BasicIOInfoStructure()
     io_info_initialize.initialize(basic_io_structure.initial())
+    ceph_conf = CephConfOp()
```

Comment:
is there any ceph.conf operations in this TC.

Reply:
No there are no ceph.conf operations in this TC.